stable_diffusion_videos/stable_diffusion_walk.py

```diff
@@ -181,12 +238,19 @@ def walk(
             if do_print_progress:
                 print(f"COUNT: {frame_index+1}/{len(seeds)*num_steps}")
 
+            imgname = output_path / ("frame%06d.jpg" % frame_index)
+            frame_index += 1
+
             if use_lerp_for_text:
                 embeds = torch.lerp(embeds_a, embeds_b, float(t))
             else:
                 embeds = slerp(float(t), embeds_a, embeds_b)
             latents = slerp(float(t), latents_a, latents_b)
 
+            if imgname.is_file():
```

Comment:
Honestly I put it here because I didn't understand it and wasn't sure if it was necessary to reach the same state as previous runs, IE if skipping would cause undesired results. I agree if you do.

Reply:
It's ok, I can move it where it needs to be, I think. Probably will take longer to write explanation than it would be to just copy paste it to right location haha.